### ISSUE TITLE
[issue]: cannot paste/ ctrl+v on windows

### ISSUE THREAD

**Issue by alexn-s** [johnlindquist/kit#965]
i just noticed that i can not paste into 
```ts
// kit = name for default import
await kit.fields([
				{ label: "top level domain", value: "de", placeholder: "de" }
			])
```
the field/ s on windows.

![image](https://user-images.githubusercontent.com/60710873/205512026-451219f4-27d9-4ac1-8c77-dbcd91af6f24.png)

in the main menu i can past things
![image](https://user-images.githubusercontent.com/60710873/205512177-aed9a45e-df74-4369-9952-b590a13f2e4e.png)

---

is there any workaround?

**Comment by alexn-s** [comment]
awesome. much appreciated

**Comment by alexn-s** [comment]
fixed by v1.39.16 (#970)
